maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/src/main/java/org/apache/maven/scm/provider/git/jgit/command/JGitUtils.java

```diff
@@ -308,7 +336,7 @@ public static List<ScmFile> addAllFiles( Git git, ScmFileSet fileSet )
 
             if ( file.exists() )
             {
-                String path = relativize( baseUri, file );
```

Comment:
this needs to have normalized backslashes

Reply:
No, all JGit commands take relative repo paths with forward slashes (independent of OS/FS) (https://archive.eclipse.org/jgit/site/4.6.0.201612231935-r/apidocs/org/eclipse/jgit/api/AddCommand.html#addFilepattern-java.lang.String-)

Reply:
Interesting, this isn't enforced upto the `PathFilter`. Maybe this should be done in relativize directly...

Reply:
This, I don't understand. It returns the path from the URI which will always be forward slash. By normalize?

Reply:
No need to normalize if it is already forward slashes.